conduit/data/datasets/audio/base.py

```diff
@@ -20,7 +20,7 @@
 __all__ = ["CdtAudioDataset"]
 
 
-class CdtAudioDataset(CdtDataset):
+class CdtAudioDataset(CdtDataset[I, npt.NDArray[np.string_], Y, S]):
```

Comment:
It seems weird to me that no method signature in this class makes use of the typevars, but maybe that's correct.